packages/types/src/documents/global/ai.ts

```diff
@@ -8,4 +8,5 @@ export interface CustomAIProviderConfig extends Document {
   model: string
   apiKey?: string
   liteLLMModelId: string
+  webSearch?: WebSearchConfig
```

Comment:
NIT: Should this be called `webSearchConfig`? Otherwise, it seems like a boolean to me.